cmake/modules/FindDeprecated.cmake

```
@@ -105,6 +105,12 @@ if("SOURCES" IN_LIST Deprecated_FIND_COMPONENTS)
   endif()
 endif()
 
+if("PRJ_BOARD" IN_LIST Deprecated_FIND_COMPONENTS)
+  list(REMOVE_ITEM Deprecated_FIND_COMPONENTS PRJ_BOARD)
```

```suggestion
  # This code was deprecated after Zephyr v3.3.0
  list(REMOVE_ITEM Deprecated_FIND_COMPONENTS PRJ_BOARD)
```

Ref:
https://github.com/zephyrproject-rtos/zephyr/blob/369ffb4929cd5a0f2253e0fdaf15b70ec275c3aa/cmake/modules/FindDeprecated.cmake#L22